src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.html

```diff
@@ -42,6 +42,10 @@
                 [readOnly]="readOnly && !isConflictingFeedbackEditable"
                 (ngModelChange)="didChange()"
             ></textarea>
+            <div class="card-header" *ngIf="!conflictMode && feedback.credits !== 0 && textBlock?.numberOfAffectedSubmissions && feedback.type === FeedbackType.MANUAL">
+                <fa-icon [icon]="'exclamation-triangle'" size="1x" class="text-warning" placement="left"></fa-icon>
+                <span jhiTranslate="artemisApp.textAssessment.feedbackImpactWarning" [translateValues]="{ affectedSubmissionsCount: textBlock.numberOfAffectedSubmissions }"></span>
```

Comment:
I think it is nice to put a text inside the span to understand the code better and as a fallback.

Reply:
Thanks for the feedback. Applied the suggestion